### PR TITLE
feat: expose base avatars by collection id

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: lambdas/package.json [hunk 1]
```diff
@@ -11,13 +11,12 @@
...
   },
   "dependencies": {
-    "@dcl/catalyst-api-specs": "^2.2.2",
+    "@dcl/catalyst-api-specs": "^3.0.0",
     "@dcl/catalyst-contracts": "^3.1.3",
     "@dcl/crypto": "^3.3.1",
     "@dcl/schemas": "^6.15.0",
     "@dcl/urn-resolver": "^2.0.3",
     "@well-known-components/env-config-provider": "^1.2.0",
-    "@well-known-components/http-server": "^1.1.6",
```
Comment (marianogoldman) [inline lambdas/package.json:20]: 🤔

Comment (marianogoldman) [reply lambdas/package.json:20]: This broke lambdas is .zone

**Review by marianogoldman — APPROVED** [review]

File: content/test/integration/controller/active-entities.spec.ts [hunk 2]
```diff
@@ -565,8 +568,9 @@ describe('Integration - Get Active Entities', () => {
 
       expect(response).toBeDefined()
       expect(response.length).toBe(1)
-      expect(response[0].entityId).toBe(secondDeploy.controllerEntity.id)
-      expect(response[0].pointer).toBe('urn:decentraland:mumbai:collections-thirdparty:athirdparty:wintercollection:1')
+      expect(response[0].pointers).toEqual(pointer.map((p) => p.toLocaleLowerCase()))
+      expect(response[0].id).toEqual(secondDeploy.controllerEntity.id)
```
Comment (menduz) [inline content/test/integration/controller/active-entities.spec.ts:572]:
```
response[0].entityId => response[0].id
```

Watch out, breaking change

Comment (pedrotambo) [reply content/test/integration/controller/active-entities.spec.ts:572]: Yeah, already checked with dApps team. Current endpoint was not used for a long time

File: content/src/controller/filter-by-urn-handler.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  context: HandlerContextWithPath<'activeEntities', '/entities/active/collections/:collectionUrn'>
+) {
+  const collectionUrn: string = context.params.collectionUrn
+
+  const parsedUrn = await isUrnPrefixValid(collectionUrn)
+  if (!parsedUrn) {
+    return {
+      status: 400,
+      body: {
+        errors: `Invalid collection urn param, it should be a valid urn prefix of a 3rd party collection, instead: '${collectionUrn}'`
```
Comment (pedrotambo) [inline content/src/controller/filter-by-urn-handler.ts:45]: or base wearables?

**Pull request merged**